docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastTree.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.Regression
+{
+    public static class FastTree
+    {
```

Comment:
>{ [](start = 4, length = 1)

So do we need a Binary and Ranking examples for FastTree?

Reply:
all the samples will come in my next PR. please also see my email for the suggested template for those samples. 

---
In reply to: [265818226](https://github.com/dotnet/machinelearning/pull/2970#discussion_r265818226) [](ancestors = 265818226)